howl/data/dataset/phone.py

```diff
@@ -0,0 +1,94 @@
+from collections import defaultdict
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Mapping, List
+import enum
+
+
+__all__ = ['PhoneEnum',
+           'PronunciationDictionary',
+           'PhonePhrase']
+
+
+class PhoneEnum(enum.Enum):
+    SILENCE = 'sil'
+    SILENCE_OPTIONAL = 'sp'
```

Comment:
what is this for exactly?

Reply:
Those are strings representing the silence and unknown speech phonemes from ARPABET and MFA.